starter-code/src/Contacts.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from "react";
+import contacts from "./contacts.json";
+
+class Contacts extends Component {
+  state = {
+    firstFive: [
```

Comment:
This is a bit hard-coding right here, it would be cleaner to just slice the first 5 contacts from your imported contacts